src/components/Forum/Forum.styles.tsx

```diff
@@ -0,0 +1,33 @@
+import { TextareaAutosize } from "@mui/base";
```

Comment:
import { TextareaAutosize } from "@mui/material";

Can we import like this ?

Reply:
Changes to import { TextareaAutosize } from "@mui/material";